src/video/resources/spaces.ts

```diff
@@ -18,32 +15,23 @@ const BROADCAST_PATH = (spaceId: string, broadcastId: string) =>
   `${SPACE_PATH(spaceId)}/broadcasts/${broadcastId}`;
 
 export class Broadcasts extends Base {
-  create(
-    spaceId: string,
-    request: CreateBroadcastRequest
-  ): Promise<BroadcastResponse> {
+  create(spaceId: string, request: CreateBroadcastRequest): Promise<Broadcast> {
     return this.http.post(`${SPACE_PATH(spaceId)}/broadcasts`, request);
   }
 
-  get(spaceId: string, broadcastId: string): Promise<BroadcastResponse> {
+  get(spaceId: string, broadcastId: string): Promise<Broadcast> {
     return this.http.get(BROADCAST_PATH(spaceId, broadcastId));
   }
 
-  delete(spaceId: string, broadcastId: string): Promise<BroadcastResponse> {
+  delete(spaceId: string, broadcastId: string): Promise<Broadcast> {
```

Comment:
I think this returns an empty response: https://docs.mux.com/api-reference/video#operation/delete-space

Reply:
whoops looks like this was there before these changes